### PR TITLE
feat: lower data receipt cost

### PR DESCRIPTION
**Pull request by Longarithm**
Main goal of this PR is to reduce data receipt cost, due to new estimation.

**Cost is reduced by 4.7 Tgas and 4.2 Ggas per KB.**
Note: the latter one is not going to have big impact, because maximal length of data receipt in mainnet is ~0.5 KB:
```
nearcore % psql postgres://public_readonly:nearprotocol@104.199.89.51/mainnet_explorer
mainnet_explorer=> select max(length(data)) as len from data_receipts;
 len
-----
 567
```
 
Secondary goal is to make protocol configs of all chains the same by default. Doesn't seem significant to make separate PR for this.

Test plan
---------
- Check that cost is lowered
- Check that genesis config is overridden for new protocol version

### CODE REVIEW

**Comment by Longarithm** [comment]
Steps to reproduce: https://hackmd.io/w6ODyKjUReuuofXTuqdyFQ

File: core/primitives/src/runtime/config_store.rs
Comment (bowenwang1996) [inline core/primitives/src/runtime/config_store.rs:21]: I think we should consider guarding this with nightly protocol feature as well so that we don't forget to change it when it is ready for release.

**Review by bowenwang1996 — APPROVED** [review]

**Review by matklad — APPROVED** [review]
Confirmed that I can reproduce the costs on my machine,  that they are not sensitive to the number of accounts (I got the same costs with 20_000 and with 1_000_000 accounts), and that they also reproduce on the new estimator. 

What I haven't confirmed *yet* is that the logic to compute the cost itself is indeed reasonable (that is, that `data_receipt_10b_1000` and `data_receipt_base_10b_1000` indeed measure what they are supposed to measure, in terms of runtime behavior). I would love to take a somewhat deeper look into that, but I don't want to block here, so LGTM if @Longarithm thinks that there aren't *logical* errors in the cost computation.

**Pull request merged**
